packages/relayer/test/Relayer.test.ts

```diff
@@ -19,6 +19,12 @@ describe('Relayer', () => {
     relayer = await deploy('Relayer', [executor.address, collector.address, owner.address])
   })
 
+  context('when an account tries to send ETH', () => {
```

Comment:
The standard function for doing this is called `receive` 

```suggestion
  describe('receive', () => {
```